cpp/include/nvtext/tokenize.hpp

```diff
@@ -112,15 +116,17 @@ std::unique_ptr<cudf::column> tokenize(
  * All null row entries are ignored and the output contains all valid rows.
  * The number of tokens for a null element is set to 0 in the output column.
  *
- * @param strings Strings column to use for this operation
- * @param delimiter Strings used to separate each string into tokens;
+ * @param input Strings column to use for this operation
```

Comment:
nit: only these 2 `count_tokens` has `Strings column to use for this operation` a bit generalised argument name. others specify what it is used for. It's a matter of personal preferance and general documentation guideline. should this be changed to more specific like `Strings column to count the tokens` ?